upup/models/cloudup/resources/addons/networking.romana/k8s-1.6.yaml.template

```diff
@@ -0,0 +1,347 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: romana-listener
+rules:
+- apiGroups:
+  - "*"
+  resources:
+  - pods
+  - namespaces
+  - nodes
+  - endpoints
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - extensions
+  resources:
+  - networkpolicies
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - "*"
+  resources:
+  - services
+  verbs:
+  - update
+  - list
+  - watch
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: romana-listener
+  namespace: kube-system
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: romana-listener
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: romana-listener
+subjects:
+- kind: ServiceAccount
+  name: romana-listener
+  namespace: kube-system
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: romana-agent
+rules:
+- apiGroups:
+  - "*"
+  resources:
+  - pods
+  - nodes
+  verbs:
+  - get
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: romana-agent
+  namespace: kube-system
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: romana-agent
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: romana-agent
+subjects:
+- kind: ServiceAccount
+  name: romana-agent
+  namespace: kube-system
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: romana-etcd
+  namespace: kube-system
+spec:
+  clusterIP: {{ .Networking.Romana.EtcdServiceIP }}
+  ports:
+  - name: etcd
+    port: 12379
+    protocol: TCP
+    targetPort: 4001
+  selector:
+    k8s-app: etcd-server
+  sessionAffinity: None
+  type: ClusterIP
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: romana
+  namespace: kube-system
+spec:
+  clusterIP: {{ .Networking.Romana.DaemonServiceIP }}
```

Comment:
Is this actually changeable?  I note that the clients aren't templated with this value, so I'm wondering how they reach the service.

Reply:
Most of the containers take the address from the environment variables that get added to the pod automatically for services. Those ones don't need to have the value passed in explicitly via flags.